lib/clustering/cluster_getter.rb

```diff
@@ -15,10 +15,7 @@ def initialize(ocns)
     end
 
     def get
-      raise "not implemented"
-      Retryable.new.run do
-        try_strategies.tap { |c| yield c if block_given? }
-      end
+      try_strategies.tap { |c| yield c if block_given? }
```

Comment:
this whole chain of `get -> try_strategies -> find_or_merge` is hard to follow, but i don't have a better solution on hand

Reply:
Agreed. Worth putting a pin in but maybe not actionable right now.

Reply:
This whole thing is **super confusing** and it works for unexpected reasons in the old (MongoDB) version...

Reply:
I rewrote this part in a way I hope is easier to follow. I am also thinking about changing `ClusterGetter.new(@ocns).get` to `ClusterOCNs.new(@ocns).cluster` to better reflect that the purpose of this class is to enforce that these OCNs are together in a cluster.

Reply:
@mwarin When you're back I'd like to discuss a couple options regarding naming here:

* `ClusterGetter.new(@ocns).get` (i.e. the same as now)
* `ClusterOCNs.new(@ocns).cluster` (renaming this class)
* `Cluster.cluster_ocns!(@ocns)` (inline into `Cluster`)

Likewise I'd like to discuss whether the names here make sense, especially `find_or_create`.

Another option would be to leave it as it is for now and to discuss/reconsider after we re-implement `#merge`.

Reply:
We will move to `Cluster.cluster_ocns!` when implementing `merge`.